app/helpers/ui_helper.rb

```diff
@@ -109,6 +109,10 @@ def print_button_to(link, options = {})
     _link_to link, { icon: "print", type: "outline-dark", text: "Print", size: "xs" }.merge(options)
   end
 
+  def medium_print_button_to(link, options = {})
+    _link_to link, { icon: "print", type: "outline-dark", text: "Print", size: "md" }.merge(options)
```

Comment:
This isn't fully necessary since you can just add `size: md` to the `options` parameter.

Reply:
Right.  Will be in next push.